pymysql/util.py

```diff
@@ -17,3 +17,18 @@ def join_bytes(bs):
         for b in bs[1:]:
             rv += b
         return rv
+
+# https://dev.mysql.com/doc/internals/en/integer.html#packet-Protocol::LengthEncodedInteger
+def lenenc_int(i):
```

Comment:
`util` module shouldn't know about MySQL protocol.
Please move this function to `connections` module.